src/screens/dashboard/MyProfile.jsx

```diff
@@ -8,19 +8,24 @@ import {
   PersonalInformation
 } from '../../components/profileFormCard/ProfileFormCard'
 import EditInformation from '../../components/editInformation/EditInformation'
-import { getMyDetails } from '../../actions/userAction'
+import { getMyDetails, getUserDetails } from '../../actions/userAction'
 import { useDispatch, useSelector } from 'react-redux'
-import { useHistory } from 'react-router-dom/cjs/react-router-dom.min'
+import { useHistory, useParams } from 'react-router-dom/cjs/react-router-dom.min'
 import VerificationModal from '../../components/verificationModal/VerificationModal'
+import { useLocation } from 'react-router-dom'
 
 function MyProfile () {
+  const { id } = useParams()
+  const { pathname } = useLocation()
   const [showEmailVerificationModal, setShowEmailVerificationModal] = useState(false)
   const [showPhoneVerificationModal, setShowPhoneVerificationModal] = useState(false)
   const history = useHistory()
   const dispatch = useDispatch()
   const userDetails = useSelector((state) => state.userDetails)
   const { status } = useSelector((state) => state.userAttrConfirmCode)
   const { user, loading } = userDetails
+  const userLogin = useSelector((state) => state.userLogin)
+  const { userInfo } = userLogin
```

Comment:
this is current logged in user and not sure this is required as it was not being sued before and getMyDetails was getting details details for user profile